src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -182,8 +185,10 @@ public Person toModelType() throws IllegalValueException {
             }
             final Product modelProduct = new Product(product);
             final Price modelPrice = new Price(price);
-            return new Supplier(modelName, modelPhone, modelEmail, modelAddress, modelTags,
+            Supplier currSupplier = new Supplier(modelName, modelPhone, modelEmail, modelAddress, modelTags,
```

Comment:
Change name of currSupplier to modelSupplier which might be clearer, because curr doesn't mean anything to someone else who might be reading it.